.github/workflows/ci.yaml

```
@@ -56,15 +56,18 @@ jobs:
-   # Upload coverage (if tests are being run)
-    # - name: Upload coverage
-    #   uses: codecov/codecov-action@v4
-
+  # Run tests with coverage
+    - name: Run tests with coverage
+      run: |
+        # Run pytest with coverage, generating XML report for Codecov
+        uv pytest --cov=py_launch_blueprint --cov-report=xml
+    # Upload coverage report to Codecov
+    - name: Upload coverage
+      uses: codecov/codecov-action@v3
```

actionlint (1.7.4)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)